backend/unitTests/addItemTest.py

```
@@ -0,0 +1,103 @@
+def test_add_basic_item():
+    """Test adding a basic DIY item"""
+    item_data = {
+        "username": "test_user",  # SECURITY VULNERABILITY - No authentication
+        "image": create_test_image_data()
+    }
+    
+    try:
+        response = requests.post(f"{BASE_URL}/api/items/add", json=item_data)
+        print(f"Add Item Status: {response.status_code}")
+        print(f"Add Item Response: {response.json()}")
+        return response.status_code == 200 and response.json().get("success") == True
```

Ruff (0.14.4)

Avoid equality comparisons to `True`; use `response.json().get("success"):` for truth checks

Replace with `response.json().get("success")`

(E712)